feat: pass required context for instructor emails (Palm)

DmytroAlipov opened the pull request:
## Description

[backport from master](https://github.com/openedx/edx-platform/pull/32625)

Pass required context to bulk enrollment emails:
- logo_url
- homepage_url
- dashboard_url

Add additional context for enrollment emails:
- contact_email
- platform_name

openedx-webhooks commented:
Thanks for the pull request, @DmytroAlipov! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.




<!-- comment:external_pr -->
<!-- data: eyJkcmFmdCI6IGZhbHNlfQ== -->

Commit:
feat: pass required context for instructor emails

pass required context to bulk enrollment emails:
  - logo_url
  - homepage_url
  - dashboard_url
add additional context for enrollment emails:
  - contact_email
  - platform_name

mphilbrick211 commented:
Hi @openedx/teaching-and-learning! Would someone be able to please review/merge this for us? Thanks!

mphilbrick211 commented:
Hi @openedx/teaching-and-learning - friendly ping on this!

mphilbrick211 commented:
> Hi @openedx/teaching-and-learning - friendly ping on this!

Hi @openedx/teaching-and-learning! Getting this back on the radar.

mariajgrimaldi commented:
@DmytroAlipov: since there will not be another release for palm, should we close this PR then?

mariajgrimaldi commented:
Hi there, @DmytroAlipov; as I mentioned before, there will not be another Palm release according to the [Open edX release schedule](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3613392957/Open+edX+Release+Schedule), so I'll close this PR in the name of BTR. You can open a backport PR to the Quince branch to include these changes in the next release or wait until Redwood. Thanks!

openedx-webhooks commented:
@DmytroAlipov Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/edx-platform&entry.867055334=https://github.com/openedx/edx-platform/pull/32742&entry.1484655318=https://github.com/DmytroAlipov&entry.752974735=2023-07-14+06:39&entry.1917517419=2024-02-23+18:41&entry.2133058324=No) so we can improve your experience in the future.

<!-- comment:end_survey -->
